andela/suyabay
# #115756483   Create an API link that leads to /developer

#### What does this PR do?
- Create an index page for developers page.
#### Description of Task to be completed?
- Create an API link that leads to /developers  route
- Design the page of the /developer which is the home page of the suyabay API 
#### How should this be manually tested?
- The user scrolls down to the footer to click the API link beside Copyright. This will direct the user to the API index page (/developer)
- In the Index page, There are 4 menus (developer, docs, partner and myApp), the user clicks any of these menus, it will direct the user to a different URL (/developer for developer, /developer/docs for docs, /developer/partners for partner and /developer/myApp for myApp).
- The different URL will display a 404 error except the developer menu. This is because the story for other menus has not been written yet.
- Check the below screenshot for a clearer picture:

![screen shot 2016-03-23 at 5 04 51 pm](https://cloud.githubusercontent.com/assets/14159420/13991688/7234490a-f119-11e5-9783-f1436f8c2ec9.png)
